app/js/lib/intent-parser/confirmation.js

```diff
@@ -0,0 +1,170 @@
+/* global TwitterCldr, TwitterCldrDataBundle */
+
+'use strict';
+
+import moment from 'components/moment';
+import 'components/cldr/en';
+import 'components/cldr/core';
+
+/*
+ * @todo:
+ *   * @see http://www.unicode.org/cldr/charts/29/verify/dates/en.html
+ *     for formatting the time of the day.
+ */
+
+const p = Object.freeze({
+  // Properties
+  listFormatter: Symbol('listFormatter'),
+
+  // Methods
+  formatUser: Symbol('formatUser'),
+  formatAction: Symbol('formatAction'),
+  formatTime: Symbol('formatTime'),
+  isToday: Symbol('isToday'),
+  isTomorrow: Symbol('isTomorrow'),
+  isThisMonth: Symbol('isThisMonth'),
+  formatHoursAndMinutes: Symbol('formatHoursAndMinutes'),
+});
+
+const DEFAULT_LOCALE = 'en';
+const PATTERNS = {
+  en: {
+    pattern: `OK, I'll remind [users] [action] [time].`,
+    formatUser: (user) => user
```

Comment:
Yeah I know. Refactoring these ones sounds like too big an effort for now. Let's revisit when/if we actually go l10n.